design-documents/graph-ql/coverage/CartPrices.graphqls

```diff
@@ -0,0 +1,57 @@
+interface CartItemInterface {
+    prices: CartItemTotals
```

Comment:
Where is the CartItemTotals defined?

Reply:
Good catch, I called the type `CartItemTotals` first and then renamed it to `CartItemPrices`, forgot to change here.